src/firefly/js/metaConvert/PartAnalyzer.js

```diff
@@ -266,6 +266,9 @@ function analyzeImageResult(part, request, table, row, fileFormat, fileOnServer,
         newReq.setTitleOptions(TitleOptions.NONE);
         override= true;
     }
+    part['additionalImageParams'] && Object.entries(part['additionalImageParams']).forEach(([k,v]) => {
+        newReq.setParam(k,v);
+    });
```

Comment:
You can simplify this to:
```js
Object.entries(part.additionalImageParams ?? {} ).forEach(([k,v]) => newReq.setParam(k,v);
```

`part['additionalImageParams']` is harder to read than `part.additionalImageParams` we rarely use that except if there is a variable in the brackets.  Using the syntax `part.additionalImageParams ?? {}` says if the object is `undefined` then use an empty object which makes an empty array which makes the forEach a noop.